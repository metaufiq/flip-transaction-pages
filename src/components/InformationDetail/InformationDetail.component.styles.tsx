import { StyleSheet } from "react-native";

const styles = StyleSheet.create({
  container: {},
  header: {
    fontWeight: 'bold'
  }
})

export default styles;