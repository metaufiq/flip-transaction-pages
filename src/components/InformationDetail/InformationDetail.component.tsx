import React from "react";
import { Text, TouchableHighlight, TouchableOpacity, View } from "react-native";

import styles from "./InformationDetail.component.styles";
import { Props } from "./InformationDetail.component.types";

const InformationDetail = ({header, content}: Props)=>{
  return(
    <View>
      <Text style={styles.header}>{header}</Text>
      <Text>{content}</Text>
    </View>
  )
}

export default InformationDetail;