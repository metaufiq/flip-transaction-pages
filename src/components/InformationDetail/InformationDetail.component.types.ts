export interface Props{
  header: string,
  content: string | number
}