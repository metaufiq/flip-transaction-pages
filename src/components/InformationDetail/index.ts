import InformationDetailCard from "./InformationDetail.component";

export default InformationDetailCard