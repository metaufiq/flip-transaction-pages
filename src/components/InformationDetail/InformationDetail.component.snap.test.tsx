import { assertSnapshots } from "../../../test";
import InformationDetail from ".";
import { Props } from "./InformationDetail.component.types";
import { Transaction } from "../../../index.types";

describe('InformationDetailSnapshot', () => {


  const mockProps:Props = {
    header: 'header',
    content: 'content'
  };

  const configs = [
    {
      props: mockProps,
      desc: 'should render information detail'
    },
  ];

  assertSnapshots(InformationDetail, configs);
});
