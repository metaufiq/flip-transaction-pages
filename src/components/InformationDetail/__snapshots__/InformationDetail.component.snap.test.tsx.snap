// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InformationDetailSnapshot should render information detail 1`] = `
<View>
  <Text
    style={
      Object {
        "fontWeight": "bold",
      }
    }
  >
    header
  </Text>
  <Text>
    content
  </Text>
</View>
`;
